react-frontend/src/components/Nav/covidbanner.js

```diff
@@ -20,6 +20,7 @@ function CovidBanner(props) {
           <Alert
             style={{
               backgroundColor: '#355992',
+              border: 'none',
```

Comment:
I've addressed this in my current PR (I think we both saw it and added the none flag.  I would greatly appreciate deleting this change to prevent merge conflicts.

Reply:
Sure, can you also put in the other commit as well to fix #408? Then we can close this PR.